package.json

```diff
@@ -70,7 +70,8 @@
     "@babel/plugin-transform-runtime": "^7.10.3",
     "@babel/preset-env": "^7.10.3",
     "@babel/runtime-corejs3": "^7.10.3",
-    "@okta/okta-auth-js": "4.6.2",
+    "@okta/okta-auth-js": "4.7.1",
+    "@okta/okta-idx-js": "0.10.0",
```

Comment:
We can use a more flexible semver for our **internal** dependencies, we have in the past. It did sometimes cause confusion because you'd have to check the yarn.lock to know the actual version which was bundled (I don't think the yarn.lock is published with the npm module like the package.json is).

Currently, pushing an update for any dependency requires re-bundling with an updated yarn.lock. (and a new published widget version)

It seems to have worked out better for us to use a specific version in package.json for our **internal** dependencies, so everyone knows exactly which version we are bundling in the widget. 


When we move the widget to ES6 / Typescript, then it is likely we will not be using webpack anymore. At that point we may declare these as **peer** dependencies, with a flexible semver indicating minimum required version.